### ISSUE TITLE
 adding icons

### ISSUE THREAD

**Issue by andsofine** [Project-1CE/vendor_cayicons#1]
will the icons requested here be added?

**Comment by alk3pInjection** [comment]
No. I'm not the author of these icons. ---- On Sun, 2022-05-29 17:51:33  ***@***.***  wrote ----
will the icons requested here be added?
